modules/core/03-connection/types/keys.go

```diff
@@ -60,3 +63,7 @@ func ParseConnectionSequence(connectionID string) (uint64, error) {
 
 	return sequence, nil
 }
+
+func GeneratedConnectionIDKey(clientID, counterpartyConnectionID string) string {
```

Comment:
Open to suggestions on a better name

Reply:
how about `ExistingConnectionIDKey` or even `ConnectionIDKey`, does the fact that it was generated in some way matter in this context?

Reply:
I like `ExistingConnectionIDKey` since its more descriptive of the problem this is solving and `ConnectionIDKey` can be confused with the ConnectionKey that stores the ConnectionEnd.

Will wait for quorum before changing

Reply:
+1 for `ExistingConnectionIDKey`

Reply:
Done